web/services/installation.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/google/uuid"
+	"github.com/trento-project/trento/web/entities"
+	"gorm.io/gorm"
+)
+
+//go:generate mockery --name=Installer --inpackage --filename=installer_mock.go
+
+type Installer interface {
+	InitializeIdentifier() error
+}
+
+type installer struct {
+	db *gorm.DB
+}
+
+func NewInstallerService(db *gorm.DB) Installer {
+	return &installer{db: db}
+}
+
+func (i *installer) InitializeIdentifier() error {
+	var trento entities.Trento
```

Comment:
Why so many new lines in this methods hehe?
It looks strange that all lines are splited XD

Jokes apart, `gorm` offers 
- https://gorm.io/docs/advanced_query.html#FirstOrInit
- https://gorm.io/docs/advanced_query.html#FirstOrCreate
Maybe some of them would make the code prettier

Reply:
I see, I'll remove some newlines :stuck_out_tongue_closed_eyes: 

In relation to `FirstOrInit` and `FirstOrCreate` I am not sure they fit the case :thinking: I might be wrong.

I understand those methods as: I give you this entity (Trento) with these values in it - in our case the InstallationID - and if it exists give it to me, otherwise create it.

The problem I see is that we don't know upfront the installation identifier we are looking for, that's what we are querying for.

I say: give me the very first top record of trento, if it's there I am done, otherwise I create a new instance, so next time it will exist and not recreate.

I tried something like
```
installationID := uuid.New()

db.FirstOrCreate(&trento, entities.Trento{InstallationID: installationID.String()})
```
but it clearly creates a new trento record all the times. That's not what we want I guess :joy: 

Again, I might be missing something about how that works.

Reply:
I see, you could create other dummy primary key with a constant value maybe hehe
Or create the keys based in some seed.
Anyway, feel free to omit this comment, the current code should do the thing

Reply:
Thanks for the tip, I'll keep those methods in mind :wink: